rubin_sim/moving_objects/ooephemerides.py

```diff
@@ -34,17 +34,17 @@ def get_oorb_data_dir():
 
 
 class PyOrbEphemerides:
-    """Generate ephemerides and propagate orbits using the python interface to Oorb.
+    """Generate ephemerides and propagate orbits,
+    using the python interface to Oorb.
 
     Typical usage:
-    pyephs = PyOrbEphemerides()
-    # Set the orbital parameters, using an lsst.sims.movingObjects.Orbits object
-    pyephs.setOrbits(orbits)
-    # Generate ephemerides at times 'times'.
-    ephs = pyephs.generateEphemerides(times, timeScale='UTC', obscode='I11')
 
-    This class handles the packing and unpacking of the fortran style arrays that
-    pyoorb uses, to and from more user-friendly pandas arrays.
+    >>> pyephs = PyOrbEphemerides()
```

Comment:
Put this in a separate "Examples" section to follow the DM standard ( https://developer.lsst.io/python/numpydoc.html#py-docstring-examples )